azext_edge/edge/providers/support_bundle.py

```diff
@@ -117,7 +111,7 @@ def build_bundle(
 
```

Comment:
Are there any CRs associated with azure arc ?

Reply:
there is one connectedcluster CR, the properties there are overlapped with the clusterconnect-agent, and I don't see much more information help diagnosis, therefore should we add that?

Reply:
talked offline, ok to exclude this CR for now